consensus/cometbft/service/abci.go

```diff
@@ -55,7 +55,7 @@ func (s *Service) PrepareProposal(
 func (s *Service) Info(context.Context,
 	*cmtabci.InfoRequest,
 ) (*cmtabci.InfoResponse, error) {
-	lastCommitID := s.sm.CommitMultiStore().LastCommitID()
+	lastCommitID := s.sm.GetCommitMultiStore().LastCommitID()
```

Comment:
I introduced the `GetCommitMultiStore` to reduce the structs implementing `CommitMultistoreAccessor`